front-end/src/ato/ato_form.jsx

```diff
@@ -21,10 +21,10 @@ const AtoForm = withFormik({
       inlet: data.inlet || '',
       one_shot: data.one_shot || false,
       disable_on_alert: data.disable_on_alert || false,
-      period: data.period || 120,
+      period: data.period || 60,
```

Comment:
why?

Reply:
sorry, this question is ambiguous.

_Why is ato stuff changed **here**?_    a result of the branching I chose, see in equipment_validation.
_Or why is ato timing changed **at all**?_    see explanation below.

Reply:
the later :-) , why change it to 120